muni_portal/core/admin.py

```diff
@@ -1,6 +1,5 @@
 from django.contrib import admin
 from muni_portal.core.models import Webhook, WebPushSubscription, WebPushNotification, WebPushNotificationResult
```

Comment:
```suggestion
from muni_portal.core.models import Webhook
```

Reply:
could you remove these models too?

Reply:
oh thanks! i'll do that and include migrations for them